Makefile

```diff
@@ -5,6 +5,30 @@ FORCE:
 ./vendor: go.mod go.sum
 	@go mod vendor
 
+.PHONY: cross
+cross: amd64 386 arm arm64 ## build cross platform binaries
+
+.PHONY: amd64
+amd64:
+	GOOS=linux GOARCH=amd64 go build -o bin/tkn-linux-amd64 ./cmd/tkn
```

Comment:
shouldn't we set the `-ldflags ` for `version`?

Reply:
@sthaha not really, those binaries are not published that way, this is there just to be able to cross-build